tests/test_bert_embedder.py

```diff
@@ -0,0 +1,234 @@
+import torch
+import pytest
```

Comment:
Was @damien2012eng  also using `pytest` or was it `nose`?

Reply:
Neither. I believe he was just using `unittest`. I've modified his code to use `pytest` too.

Reply:
Perfect! Would it be possible to set up github actions to quickly run these tests automatically for every branch/PR? We can reuse @tamarl08's setup from the `match` repository?

Reply:
That's a good idea! Let me take a look!